corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -280,11 +280,15 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
         },
 
         templateContext: function () {
-            var appId = utils.currentUrlToObject().appId;
+            var appId = utils.currentUrlToObject().appId,
+                md = window.markdownit();
             return {
                 data: this.options.model.get('data'),
                 styles: this.options.styles,
                 isMultiSelect: this.options.isMultiSelect,
+                renderMarkdown: function (datum) {
+                    return md.render(DOMPurify.sanitize(datum || ""));
```

Comment:
Do I get that right that `md.render` does not support html in markdown? Because I think some markdown interpreters do allow for that. Not sure if `<script>` would be one of them, which would be a problem

Reply:
yeah, that's right, this particular library escapes any HTML it's given:
```
> window.markdownit().render("## abc")
'<h2>abc</h2>\n'

> window.markdownit().render("## a<i>b</i>c")
'<h2>a&lt;i&gt;b&lt;/i&gt;c</h2>\n'
```